Dockerfile

```
@@ -1,7 +1,7 @@
-FROM jinaai/jina:2.0.0rc7
+FROM jinaai/jina:2.0.5
```

Let's update it to the latest release
```suggestion
FROM jinaai/jina:2.0.7
```

---

updated